docs/src/main/sphinx/routines/loop.md

```diff
@@ -24,27 +24,34 @@ block](routine-label).
 
 ## Examples
 
-
-The following function counts up to `100` in a loop starting from the input
-value `i` and returns the number of incremental steps in the loop to get to
-`100`.
+The following function counts up to `100` with a step size `step` in a loop
+starting from the start value `i`, and returns the number of incremental steps
+in the loop to get to a value of `100` or higher:
 
 ```sql
-FUNCTION to_one_hundred(i int)
+FUNCTION to_one_hundred(i int, step int)
```

Comment:
`start` might be better than `i`.

Reply:
start is a keyword .. adapted to use `start_value` and then an internal value `current` .. and tested it all again. Good to go now.